frontend/src/app/main/ui/workspace/colorpalette.cljs

```diff
@@ -160,10 +160,16 @@
 
      [:span.left-arrow {:on-click on-left-arrow-click} i/arrow-slide]
      [:div.color-palette-content {:ref container :on-wheel on-scroll}
-      [:div.color-palette-inside {:style {:position "relative"
+     (if (empty? current-colors)
+        [:div.color-palette-empty {:style {:position "absolute"
+                                           :left "50%"
+                                           :top "50%"
+                                           :transform "translate(-50%, -50%)"}} 
+              "There are no Color Styles in your Library yet"]
```

Comment:
This should be translated, with something similar that what it do in line 139 of this file.

You should use the tr function with a key. That key should exist in frontend/translations/en.po